app/assets/stylesheets/enhanced_fixed.css.scss

```diff
@@ -1,3 +1,4 @@
 $responsive: false;
 
 @import 'shared/enhanced';
+@import 'moneyhelper_styles';
```

Comment:
We should import the `reskin/all` styles in this stylesheet as well as in `enhanced_responsive`.

Reply:
Resolved